traffic-control-openshift/README.md

```diff
@@ -1,25 +1,59 @@
 # Traffic Control on OpenShift
 This demo demonstrates how to use traffic control to test applications running in [OpenShift](https://www.openshift.com/). It uses OpenShift version 3 to run the Kubernetes cluster, [tcd](https://github.com/kinvolk/tcd) to do traffic control on the Kubernetes nodes and [Weave Scope](https://github.com/weaveworks/scope) to monitor, visualize and interact with the Kubernetes cluster.
 
-## What do you need to start?
+## Prepare for the demo
+### What do you need to start?
 - Get an [OpenShift ready VM](https://github.com/kinvolk/openshift-evangelists-vagrant-origin) to run the demo. You will need to reboot the VM after the first `vagrant up` in order to load the right kernel version and to load the modules needed by tcd.
 - Get the [demo](https://github.com/kinvolk/demo) repository
 
-You can reach the OpenShift dashboard by pointing your browser to [https://10.2.2.2:8443](https://10.2.2.2:8443). Use user 'admin' and password 'admin' to login.
+### Get the VM
+```
+git clone https://github.com/kinvolk/openshift-evangelists-vagrant-origin.git
+cd openshift-evangelists-vagrant-origin
+vagrant up
+```
+The building process could take several minutes (15 minutes) depending of your machines and the output should look like this:
+
+```
+==> default: You can now access OpenShift console on: https://10.2.2.2:8443/console
+==> default: To use OpenShift CLI, run:
+==> default: $ vagrant ssh
+==> default: $ sudo -i
+==> default: $ oc status
+==> default: $ oc whoami
+==> default: If you have the oc client library on your host, you can also login from your host.
+==> default: $ oc login https://10.2.2.2:8443
+```
+To reboot you can do
+```
+vagrant halt; vagrant up
+```
 
-For convenience, you can add this to your ~/.ssh/config the output of ```vagrant ssh-config``` and add the following lines.
+For convenience, you can add this to your ~/.ssh/config the output of ```vagrant ssh-config``` using as host name ```openshift``` and add the following lines.
 
 ```ssh
      LocalForward 1931 172.30.8.64:4040
      LocalForward 8082 172.30.8.92:80
      LocalForward 8083 172.30.8.93:80
 ```
 
-This will allow you to use this [http://127.0.0.1:1931](http://127.0.0.1:1931) to access the Weavescope dashboard.
+This will allow you to access to the Weavescope dashboard from [http://127.0.0.1:1931](http://127.0.0.1:1931).
```

Comment:
Keep a `ssh openshift` session open and the ssh port forwarding will allow you to...